modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -70,7 +95,7 @@ Example DataWeave 2.0 expression to be used when configuring the policy:
 #[attributes.headers.['client_secret']]
 ----
```

Comment:
In this example the policy is configured to expect two headers: client_id and client_secret with the pair of credentials. The policy is flexible to allow other type of headers as well. 

This is the default configuration for the policy.

Reply:
Done! Thanks!